### PR TITLE
chore: prepare for major version bump

### PR DESCRIPTION
**Pull request by igorbernstein2**
This updates all next versions to be 2.0.0-alpha-1

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java [hunk 1]
```diff
@@ -30,18 +30,19 @@
...
     assertTrue("incorrect version format", versionMatcher.matches());
 
     int result =
         ComparisonChain.start()
-            .compare(1, Integer.parseInt(versionMatcher.group(1)))
-            .compare(14, Integer.parseInt(versionMatcher.group(2)))
-            .compare(1, Integer.parseInt(versionMatcher.group(3)))
+            .compare(2, Integer.parseInt(versionMatcher.group(1)))
+            .compare(0, Integer.parseInt(versionMatcher.group(2)))
+            .compare(0, Integer.parseInt(versionMatcher.group(3)))
```
Comment (kolea2) [inline bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java:42]: let's add in a small assertion for the `-alpha`

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java (hunk 1 above)
Comment (igorbernstein2) [reply bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java:42]: I dont want to update this test for every release. It's just meant as a sanity test that the current version we are releasing is > 2.0.0

Comment (igorbernstein2) [reply bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java:43]: I updated the test to pattern match the suffix as well

File: bigtable-dataflow-parent/bigtable-beam-import/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Cloud Dataflow.
 
 ## Instructions
 [//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
-Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.14.1/bigtable-beam-import-1.14.1-shaded.jar), which is an aggregation of all required jars.
+Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/2.0.0-alpha-1/bigtable-beam-import-2.0.0-alpha-1-shaded.jar), which is an aggregation of all required jars.
```
Comment (kolea2) [inline bigtable-dataflow-parent/bigtable-beam-import/README.md:8]: let's revert README changes. release please should update them for us as part of the actual release

**Review by kolea2 — APPROVED** [review]
lgtm after readme cleanup

File: bigtable-dataflow-parent/bigtable-beam-import/README.md (hunk 2 above)
Comment (igorbernstein2) [reply bigtable-dataflow-parent/bigtable-beam-import/README.md:8]: oh, good catch! not sure how those snuck in there

**Pull request merged**